drizzlepac/buildmask.py

```diff
@@ -81,7 +81,7 @@ def buildDQMasks(imageObjectList,configObj):
 
 def buildMask(dqarr, bitvalue):
     """ Builds a bit-mask from an input DQ array and a bitvalue flag """
-    return bitfield_to_boolean_mask(dqarr, bitvalue, good_mask_value=1,
+    return bitfield_to_boolean_mask(dqarr.astype(np.int16), bitvalue, good_mask_value=1,
```

Comment:
Why are you modifying `buildMask` in a PR that deals with PHOTUTILS. But rebase it agains the new `master` because `buildmask.py` now uses `astropy.nddata.bitmask` module.

Reply:
Hm... no need to rebase - it looks like there are no conflicts.